### DISCUSSION TITLE
Enhancement: Support for different aspect ratios?

### DISCUSSION THREAD

**Discussion by Langoblian** [FunkinCrew/Funkin#1656] [Ideas]
I think it would be pretty neat if the game supported different aspect ratios natively. I have a 4:3 CRT that I'd like to play this game on, and i'm certain that there are other fans with wider/smaller monitors they'd like to play this game on.

**Comment by ghost** [comment]
Change this to different aspect ratio support, would like to play fullscreen on my 16:10 monitor

**Comment by NotHyper-474** [comment]
I also have a 4:3 aspect ratio (1024x768) monitor, recent versions of the game with the automatic zoom display incorrectly (for instance, the HUD in a song ends up zooming too much) like in this screenshot
![image](https://user-images.githubusercontent.com/40342021/118156286-90a1a380-b3ef-11eb-9b18-be53c26cad59.png)

**Comment by Langoblian** [comment]
I honestly didn't even know there was automatic zoom, it just letterboxes on the most recent release for me, do you have to compile it yourself to get this feature?

**Comment by Langoblian** [comment]
I honestly didn't even know there was automatic zoom, it just letterboxes on the most recent release for me, do you have to compile it yourself to get this feature?
